test_runner/src/main/kotlin/ftl/config/FtlConstants.kt

```diff
@@ -52,12 +51,13 @@ object FtlConstants {
     const val applicationName = "Flank"
     const val GCS_PREFIX = "gs://"
     const val runTimeout = "-1"
+
     val JSON_FACTORY: JsonFactory by lazy { Utils.getDefaultJsonFactory() }
 
-    val bugsnag = Bugsnag(if (useMock) null else "3d5f8ba4ee847d6bb51cb9c347eda74f")
+    val bugsnag = initBugsnag(useMock)
```

Comment:
Maybe will be better to get rid of bugsnag reference and initialization in `FtlConstants` and move it to somewhere else. Also I am not sure if we need to keep reference to bugsnag because it's used only to setting app version. Additionally make sure that it will be initialized after `MockServer` start because it changes `useMock` flag. BTW I don't know why useMock flag belongs to FtlConstants, it is not a constant :), but I guess it's another story.